workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/WorkerGenerationJob.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.IO;
+using Improbable.CodeGeneration.FileHandling;
+using Improbable.CodeGeneration.Jobs;
+
+namespace Improbable.Gdk.CodeGenerator
+{
+    public class WorkerGenerationJob : CodegenJob
+    {
+        private readonly string relativeOutputPath = "improbable";
+        private readonly string package = "Improbable";
```

Comment:
This is defined here - and only passed through into the Generator and the tt file. Why not just inline it in the `tt` file?